### DISCUSSION TITLE
Help in accessing Azure pack from GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by here-sabs** [community/community#140376] [GitHub Education]
### Hi everyone,

Hey!
I'm verified for student developer pack but unable to use the  academic benefits offers .Help me to get my access .It's been a long time from getting approved of pack but I haven't received my academic benefits yet. Today I sign in for azure but it showing that I'm not verified to use this offer 

![IMG-20241002-WA0000](https://github.com/user-attachments/assets/4068b9fe-cd8f-449d-be62-3d4d2b4f0aa9)

![IMG-20241002-WA0001](https://github.com/user-attachments/assets/2755873b-f8f1-42ab-bd73-c94bd61ca46d)

**Comment by davevad93** [comment]
Hi @here-sabs ,

This is a known issue, #131336, #135739, to solve it you can do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

The users in the discussions I provided above solved it by doing that.
